### DISCUSSION TITLE
Universal Calculator

### DISCUSSION THREAD

**Discussion by Felix-E-S** [Felix-E-S/T24S2T1EXTG08#1] [General]
G'Day,

Lets get this assignment going!
I will create the title page for the Calculator by tomorrow(21/08/24) evening.

Cheers,
Felix

**Comment by StutzTAFE** [comment]
Hi Felix

I'm happy to hear it. 
Sorry for my absence.
I'll be working a lot on the Calculator in the following days to catch up.

Thanks,
Jethro

**Comment by Felix-E-S** [comment]
No worries,

We'll get it done.
